frontend/pages/register.vue

```diff
@@ -0,0 +1,261 @@
+  }
+})
+
+const canSubmit = computed(() => email.value && password1.value && password2.value && agreement.value && !Object.keys(errors.value).length)
+const addressError = ref<string|undefined>(undefined)
+const password1Error = ref<string|undefined>(undefined)
+const password2Error = ref<string|undefined>(undefined)
+const agreementError = ref<string|undefined>(undefined)
+</script>
+
+<template>
+  <BcPageWrapper>
+    <div class="content">
+      <div class="caption">
+        <div class="title">
+          Sign up to beaconcha.in
+        </div>
+        <div class="purpose">
+          to manage and monitor your validators.
+        </div>
+      </div>
+      <div class="container">
+        <form @submit="onSubmit">
+          <div class="input-row">
+            <label for="email" class="label">{{ $t('login_and_register.email') }}</label>
+            <InputText
+              id="email"
+              v-model="email"
+              type="text"
+              :class="{ 'p-invalid': errors?.email }"
+              aria-describedby="text-error"
+              @focus="addressError = undefined"
+              @blur="addressError = errors?.email"
+            />
+            <div class="p-error">
+              {{ addressError || '&nbsp;' }}
+            </div>
+          </div>
+          <div class="input-row">
+            <label for="password1" class="label">{{ $t('login_and_register.choose_password') }}</label>
+            <InputText
+              id="password1"
+              v-model="password1"
+              type="password"
+              :class="{ 'p-invalid': errors?.password1 }"
+              aria-describedby="text-error"
+              @focus="password1Error = undefined"
+              @blur="password1Error = errors?.password1"
+            />
+            <div class="p-error">
+              {{ password1Error || '&nbsp;' }}
+            </div>
+          </div>
+          <div class="input-row">
+            <label for="password2" class="label">{{ $t('login_and_register.confirm_password') }}</label>
+            <InputText
+              id="password2"
+              v-model="password2"
+              type="password"
+              :class="{ 'p-invalid': errors?.password2 }"
+              aria-describedby="text-error"
+              @focus="password2Error = undefined"
+              @blur="password2Error = errors?.password2"
+            />
+            <div class="p-error">
+              {{ password2Error || '&nbsp;' }}
+            </div>
+          </div>
+          <div class="input-row">
+            <div class="agreement">
+              <Checkbox
+                v-model="agreement"
+                input-id="agreement"
+                :binary="true"
+                class="checkbox"
+                @focus="agreementError = undefined"
+                @blur="agreementError = errors?.agreement"
+              />
```

Comment:
**Bug:** The checkbox is too big (as discussed with @Buttaa ).

Reply:
Mmmh it seems that there are differences of tastes between me, Butta and Philip.
I contacted Butta and we will ask Philip again if needed.

For now, I do not change it, but will do it if we decide so, in a mini PR later.